pkg/playground/react-patterns.js

```diff
@@ -22,8 +22,8 @@ require([
     "react",
     "performance/dialog-view",
     "playground/react-demo-dialog",
-    "base1/cockpit-components",
-], function(cockpit, React, dialog_pattern, demo_dialog, components) {
+    "base1/cockpit-components-terminal",
+], function(cockpit, React, dialog_pattern, demo_dialog, cockpit_terminal) {
```

Comment:
Oh, I thought that was what we agreed upon. Maybe I misunderstood.

I like the idea of having a cockpit component package that has everything. Makes it easier for consumers of the API.